src/bin/inx-chronicle/api/stardust/explorer/responses.rs

```diff
@@ -102,10 +102,18 @@ pub struct MilestonesResponse {
 
 impl_success_response!(MilestonesResponse);
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct BlockPayloadTypeDto {
+    pub block_id: String,
+    #[serde(rename = "type")]
```

Comment:
The issue asks for `payloadType`. Any reason you didn't use that?

Reply:
I wanted to use the same encoding that we use in the Core API.

Reply:
I think just `type` would be confusing here as it could be interpreted as if the blocks itself had a type. So I am renaming to `payloadType`.